src/layout/Header/Header.tsx

```diff
@@ -1,9 +1,124 @@
-import { HeaderWrapper } from './Header.styled';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import { StyledToolBar } from './Header.styled';
+import IconButton from '@mui/material/IconButton';
+import { Avatar, Button, Menu, MenuItem, Typography } from '@mui/material';
+import LightModeRoundedIcon from '@mui/icons-material/LightModeRounded';
+import DarkModeRoundedIcon from '@mui/icons-material/DarkModeRounded';
+import DirectionsBusRoundedIcon from '@mui/icons-material/DirectionsBusRounded';
+import testProfile from '../../assets/person1.jpeg';
+import { useState } from 'react';
+
+const settings = ['My bookings', 'Logout'];
 
 export const Header = () => {
+    const [themeBool, setThemeBool] = useState(true);
+    const [isLoginClicked, setLoginClick] = useState(false);
+    const [anchorElUser, setAnchorElUser] = useState<null | HTMLElement>(null);
+
+    const handleLoginClick = () => {
+        setLoginClick(true);
+    };
+    const handleThemeClick = () => {
+        setThemeBool(!themeBool);
+    };
+    const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorElUser(event.currentTarget);
+    };
+    const handleCloseUserMenu = (setting: string) => {
+        setAnchorElUser(null);
+        if (setting === 'Logout') {
+            setLoginClick(false);
+        }
+    };
     return (
-        <HeaderWrapper>
-            <h2>Header</h2>
-        </HeaderWrapper>
+        <>
+            <AppBar position="sticky">
+                <StyledToolBar>
+                    <IconButton
+                        size="large"
+                        edge="start"
+                        color="inherit"
+                        aria-label="menu"
+                        sx={{ mr: 2 }}
+                    >
+                        <DirectionsBusRoundedIcon
+                            sx={{ marginRight: 1, marginTop: 0.25 }}
+                        />
+                        Bustle
```

Comment:
Reduce text size. Icon should be slightly bigger or equal in size to the button text.